### ISSUE TITLE
rewrite all the AI generated code

### ISSUE THREAD

**Issue by dumorando** [mariocraft987/bark-coding#249]
me and @breakfast-for-dinner are gonna work on this

(or atleast, i think)
![image](https://github.com/mariocraft987/bark-coding/assets/144301348/f872f1a7-6023-4e43-b304-4a0f7b6eb60c)

**Comment by breakfast-for-dinner** [comment]
yeah sure. even `sketch.js`? we might have to do a full editor rewrite

**Comment by dumorando** [comment]
yea skecth.js is the worst one

**Comment by rubiidev18alt** [comment]
have we gotten to this with more than just 1 commit

**Comment by sillybreakfast** [comment]
> have we gotten to this with more than just 1 commit

what do you mean?

**Comment by dumorando** [comment]
bark is dead
